integrations/cats/src/test/scala/sttp/tapir/integ/cats/ExampleFunctorLawSpec.scala

```diff
@@ -0,0 +1,26 @@
+package sttp.tapir.integ.cats
+
+import cats.Eq
+import cats.implicits._
+import cats.laws.discipline.FunctorTests
+import org.scalacheck.Arbitrary
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatestplus.scalacheck.Checkers
+import org.typelevel.discipline.scalatest.FunSuiteDiscipline
+import sttp.tapir.EndpointIO.Example
+import sttp.tapir.integ.cats.instances._
+
+class ExampleFunctorLawSpec  extends AnyFunSuite with FunSuiteDiscipline with Checkers {
+  implicit def eqTree[A: Eq]: Eq[Example[A]] = Eq.fromUniversalEquals
```

Comment:
This will actually ignore `Eq[A]`, fyi. You can use `Eq.by(Example.unapply(_).get)` or `Eq.by(e => (e.value, e.name, e.summary))` instead :)

Also, Tree -> Example